pkg/services/publicdashboards/service/query.go

```diff
@@ -257,6 +258,10 @@ func groupQueriesByPanelId(dashboard *simplejson.Json) map[int64][]*simplejson.J
 					query.Set("datasource", datasource)
 				}
 
+				if cacheTTLOverride > 0 {
+					query.Set("queryCachingTTL", cacheTTLOverride)
```

Comment:
You've already set the queryCachingTTL on the queries in DataSourceWithBackend component. Is this needed?

Reply:
Yes - for pubdash, we don't send any more query metadata than is necessary, so we have to grab the field out of the panel json on the backend

Reply:
on second thought... it should probably be sent with the pubdash query payload. Let me see if that's feasible

Reply:
Done on [6cf32a6](https://github.com/grafana/grafana/pull/61968/commits/6cf32a68d4cdc257795a3a510f813d7c69945b67) - still requires changes on the pubdash side but things are consolidated now at least